build.gradle

```
@@ -3,15 +3,15 @@ apply plugin: 'com.gladed.androidgitversion'
 
 buildscript {
     ext {
-        kotlin_version = "1.3.72"
+        kotlin_version = "1.4.21"
 
         android_compile_sdk = 29
         android_target_sdk = 29
```

Might be a good idea to upgrade to API 30:
```suggestion
        android_compile_sdk = 30
        android_target_sdk = 30
```